projects/cds/src/profiletag/profile-tag.module.ts

```diff
@@ -1,7 +1,18 @@
+import { HTTP_INTERCEPTORS } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import { ProfileTagCmsModule } from './cms-components/profile-tag-cms.module';
+import { ConsentReferenceInterceptor } from './http-interceptors/consent-reference-interceptor';
+import { DebugInterceptor } from './http-interceptors/debug-interceptor';
 
 @NgModule({
   imports: [ProfileTagCmsModule],
+  providers: [
+    {
+      provide: HTTP_INTERCEPTORS,
+      useClass: ConsentReferenceInterceptor,
```

Comment:
Please use `useExisting` instead of `useClass` as it helps with the extensibility: https://angular.io/guide/dependency-injection-providers#aliased-class-providers